### DISCUSSION TITLE
Cron Schedule: Run a job on the first tuesday of every month

### DISCUSSION THREAD

**Discussion by sburkulemb** [dagster-io/dagster#20981] [Q&A]
Hello, I'm looking to set up a task that executes on the first Tuesday of each month. Is there a way to accomplish this? The only solution I've considered so far involves setting up a cron job to run every Tuesday, and then incorporating a check within the script to determine if it's the first Tuesday before proceeding with the task execution.

**Comment by garethbrickman** [comment]
I suspect this is a limitation of the cron spec that it's not aware of the relative position of a particular day in a month so I think you'd need a script that can be aware of the calendar dates.
